client/data/marketplace/use-es-query.ts

```diff
@@ -41,8 +42,8 @@ const mapIndexResultsToPluginData = ( results: ESHits ): Plugin[] => {
 	}
 	return results.map( ( { fields: hit, railcar } ) => {
 		const plugin: Plugin = {
-			name: hit.plugin?.title, // TODO: add localization
-			slug: hit.slug,
+			name: decodeEntities( hit.plugin?.title ), // TODO: add localization
+			slug: decodeEntities( hit.slug ),
```

Comment:
Do we need to make any changes to a slug?

Reply:
>  I took the approach from the following method

Yeah, if this is applied somewhere else I don't see a reason to not have it here too.

>  I don't expect any slug to have encoded-html characters

Me neither, but for the sake of keeping the same behavior, I don't see a reason to remove it.